azurerm/internal/services/databricks/databricks_workspace_resource.go

```diff
@@ -79,46 +79,98 @@ func resourceDatabricksWorkspace() *pluginsdk.Resource {
 				MaxItems: 1,
 				Elem: &pluginsdk.Resource{
 					Schema: map[string]*pluginsdk.Schema{
-						"no_public_ip": {
-							Type:     pluginsdk.TypeBool,
-							ForceNew: true,
+						"aml_workspace_id": {
```

Comment:
could we expand aml and no use an abbreviation?

Reply:
Fixed